packages/integrations/stores/gcp-secret-manager-config-store/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@configu-integrations/aws-parameter-store",
+  "version": "0.0.0",
+  "license": "Apache-2.0",
+  "author": "@configu/dev",
+  "files": [],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/configu/configu",
+    "directory": "packages/integrations"
+  },
+  "bugs": {
+    "url": "https://github.com/configu/configu/issues",
+    "email": "dev@configu.com"
+  },
+  "homepage": "https://configu.com",
+  "keywords": [
+    "cfgu",
+    "configu"
+  ],
+  "scripts": {
+    "build": "tsup"
+  },
+  "dependencies": {
+    "@aws-sdk/client-ssm": "^3.667.0",
```

Comment:
i don't think you need the aws-sdk on gcp integration.

Reply:
I think you are right.
should there be a dependency related to gcp to add?

Reply:
yes, this one:
```import { SecretManagerServiceClient } from '@google-cloud/secret-manager';```